pkg/network/encoding/encoding_test.go

```diff
@@ -481,6 +483,8 @@ func TestHTTPSerializationWithLocalhostTraffic(t *testing.T) {
 			): &httpReqStats,
 		},
 	}
+	// ignore "declared but not used"
```

Comment:
why is this "declared but not used"? It's used 3 lines up?

Reply:
that confused me, too. I suspect it needs to be used as more than an output arg.